app-arch/gcab/gcab-0.7.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Library and tool for working with Microsoft cabinet archives"
+DESCRIPTION="gcab is command line tool that can list, extract and create \
+cabinet (.cab) archives. The Cabinet format is often used on Microsoft Windows \
+but it can also be used on other systems with open source software such as \
+cabextract, lcab and gcab. Unlike cabextract and lcab, which can only list \
+and extract cabinet archives, gcab can also create them.
+gcab also includes a GObject shared library, libgcab, which provides all the \
+functionality available to the command line tool."
+HOMEPAGE="https://wiki.gnome.org/msitools"
+COPYRIGHT="2012-2016 Marc-André Lureau"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/gcab/$portVersion/gcab-$portVersion.tar.xz"
+CHECKSUM_SHA256="a16e5ef88f1c547c6c8c05962f684ec127e078d302549f3dfd2291e167d4adef"
+PATCHES="gcab-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	gcab$secondaryArchSuffix = $portVersion
+	cmd:gcab$commandSuffix = $portVersion
+	lib:libgcab_1.0$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix >= 0.4600.2
+	lib:libintl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gcab${secondaryArchSuffix}_devel = $portVersion
+	devel:libgcab_1.0$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	gcab$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0$secondaryArchSuffix >= 0.4600.2
+	devel:libintl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libintl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix >= 0.4600.2
+	devel:libgettextlib$secondaryArchSuffix >= 0.14.1
+	intltool >= 0.40
+	"
+BUILD_PREREQUIRES="
+	intltool
+	gettext >= 0.19.6
```

Comment:
These should be avoided when possible, they provide packages, not commands required by the build.